core/src/main/java/software/amazon/awssdk/core/sync/RequestBody.java

```diff
@@ -142,7 +154,7 @@ public static RequestBody empty() {
      * Creates a {@link RequestBody} using the specified bytes (without copying).
      */
     private static RequestBody ofByteDirect(byte[] bytes) {
-        return new RequestBody(new ByteArrayInputStream(bytes), bytes.length);
```

Comment:
Is this the appropriate mime type for bytes? I would think octet stream for this except for the of(String) which should be text/plain with UTF 8 as the charset.

Reply:
Nope, updated it to be `application/octet-stream`